test/e2e-test/pages/common/common.ts

```diff
@@ -1,7 +1,9 @@
 import * as _ from 'lodash';
```

Comment:
This file I guess was intended to include all those methods that are in common for the whole app, so we should only have one common file, dont see the need of creating an extra folder for common.